qiskit_nature/mappers/second_quantization/logarithmic_mapper.py

```diff
@@ -0,0 +1,199 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Logarithmic Mapper."""
+
+import operator
+
+from fractions import Fraction
+from functools import reduce
+from typing import List, Union
+
+import numpy as np
+
+from qiskit.opflow import PauliSumOp
+from qiskit.quantum_info.operators import SparsePauliOp, Operator
+from qiskit_nature.operators.second_quantization import SpinOp
+from .spin_mapper import SpinMapper
+
+
+class LogarithmicMapper(SpinMapper):
+    """A mapper for Logarithmic spin-to-qubit mapping [1].
+
+
+    [1]: Mathis, S. V., Gauglielmo, M., & Ivano, T. (2020).
+         Toward scalable simulations of Lattice Gauge Theories on quantum computer.
+         `arXiv:2005.10271 <https://arxiv.org/abs/2005.10271>`_
```

Comment:
Besides the misspelled name, I would also suggest to use the more up-to-date reference: https://journals.aps.org/prd/abstract/10.1103/PhysRevD.102.094501

Reply:
Could we add something in the class docstring that briefly describes the embedding in a larger matrix, I see something in the map function but when I look  at the constructor it talks about embed_padding and embed_location and there is no immediate text to put these into context here for the reader. And I wonder if simply `padding` and `location` would be easier names if the text describes these as referring to the embedded matrix.